cmd/bucket-replication-utils.go

```diff
@@ -699,3 +701,25 @@ func newBucketResyncStatus(bucket string) BucketReplicationResyncStatus {
 		Version:    resyncMetaVersion,
 	}
 }
+
+var contentRangeRegexp = regexp.MustCompile(`bytes ([0-9]+)-([0-9]+)/([0-9]+|\\*)`)
+// parse size from content-range header
+func parseSizeFromContentRange(h http.Header) (sz int64, err error) {
```

Comment:
It is little bit confusing to return 0 and -1 sometimes for an error

Reply:
When Content-Range has a `*` ,it is interpreted as size unknown, hence -1 is returned. Caller should check error returned before using the size